src/gui/properties/piecesmapwidget.cpp

```diff
@@ -0,0 +1,195 @@
+/*
+ * Bittorrent Client using Qt and libtorrent.
+ * Copyright (C) 2006  Christophe Dumez <chris@qbittorrent.org>
```

Comment:
Incorrect copyright.

Reply:
I did not found any information about this. Can you be more precise ?

Reply:
> 
> 
> I did not found any information about this. Can you be more precise ?

Aren't you the author of this file? Or did you borrow this code?
In any case, it doesn't look like Christophe Dumez wrote it. You should specify the real author(s).

Reply:
I copy/pasted another class from this project to keep file structure. But yes you're right, I'll fix that.